.vscode/launch.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Is there a reason this file should be in the repository? Although vscode is probably the best editor to use to do this work, we should not require it.